css/about.css

```diff
@@ -126,101 +128,103 @@ hr {
 .d-none {
   display: none;
 }
+
 .menu_mobile {
   display: flex;
   margin: 20px 10px 0;
 }
 
 .list-mobile {
-    width: 100%;
-    height: 100vh;
-    background: #f0f0f0;
-    display: none;
+  width: 100%;
+  height: 100vh;
+  background: #f0f0f0;
+  display: none;
 }
 
 .list-mobile.active {
-    display: flex;
+  display: flex;
 }
 
 .list-mobile ul {
-    list-style: none;
-    display: flex;
-    flex-direction: column;
-    align-items: flex-start;
-    position: absolute;
-    left: -50px;
-    top: 45px;
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  position: absolute;
+  left: -50px;
+  top: 45px;
 }
 
 .list-mobile ul li {
-    margin: 10px 12px;
-    font-size: 30px;
-    padding: 10px 12px;
+  margin: 10px 12px;
+  font-size: 30px;
+  padding: 10px 12px;
 }
 
 .list-mobile ul li a {
-    text-decoration: none;
-    color: #272a31 ;
+  text-decoration: none;
+  color: #272a31;
 }
 
 .close {
-    font-size: 25px;
-    font-family: Verdana, Geneva, Tahoma, sans-serif;
-    position: absolute;
-    left: 60vh;
-    top: 9.9px;
-    cursor: pointer;
+  font-size: 25px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  position: absolute;
+  left: 60vh;
+  top: 9.9px;
+  cursor: pointer;
 }
 
 @media only screen and (min-width: 768px) {
-    .menu_mobile {
-      display: none;
-    }
-
-    .nav-bar {
-      justify-content: space-between;
-      align-items: center;
-      width: 100%;
-      background-color: #ffffff;
-      font-size: 22px;
-      display: flex;
-    }
-
-    .nav-bar p {
-      justify-content: start;
-      margin: 12px 10px;
-    }
-
-    .nav-list {
-      list-style: none;
-      margin: 12px 10px;
-      justify-content: space-between;
-      width: 50%;
-    }
-
-    .nav-list li a {
-      text-decoration: none;
-      color: #ec5242;
-    }
-
-    .social-nav {
-      display: flex;
-      background-color: #41434b;
-      justify-content: flex-end;
-    }
-
-    .section-social a {
-      text-decoration: none;
-      color: #f0f0f0;
-    }
-
-    .section-social {
-      margin: 20px 8px 8px;
-      width: 20%;
-      justify-content: space-between;
-    }
-
-    .imgs {
-      flex-direction: row;
-    }
+  .menu_mobile {
+    display: none;
+  }
+
+  .nav-bar {
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    background-color: #fff;
+    font-size: 22px;
+    display: flex;
+    height: 85px;
+  }
+
+  .nav-bar p {
+    justify-content: start;
+    margin: 12px 10px;
+  }
```

Comment:
- [ ] This section of the page was pointed out by the previous reviewer requesting that we at least match it as much as possible with the design. I think working on the spacings can help here. Our great undoing here is 
```CSS
justify-content: space-between;
```

When we use `space-between`, we spread the elements across the viewport when that is not the case with the original design. There is spacing to the right and to the left of this section.

# Comparison
--------------------------------------Original-----------------------------------------------
![nav1](https://user-images.githubusercontent.com/55635977/128219460-c76b94d7-a3c2-43f6-8217-4ca8bf21f1ec.png)

--------------------------------------Yoursl-----------------------------------------------
![Capture](https://user-images.githubusercontent.com/45002654/128385038-16d3267c-e638-4072-93aa-7ac3e49e5642.PNG)


_Originally posted by @Emmyn5600 in https://github.com/xJokeer/Capstone_Microverse/pull/1#discussion_r682780124_

Reply:
This issue is not properly solved 😿